src/user-guide/running-workflows/workflow-completion.rst

```diff
@@ -4,25 +4,29 @@ Workflow Completion
 ===================
 
 If there is nothing more to run (according to the graph) and there are no
-:term:`incomplete tasks <incomplete task>` present, the scheduler will report
-workflow completion and shut down when current active tasks finish. 
+:term:`incomplete <output completion>` tasks present in :term:`n=0 <n-window>`
+the workflow is complete and the scheduler will shut down automatically. 
 
 
 .. _scheduler stall:
 
 Scheduler Stall
 ===============
 
-If there is nothing more to run (according to the graph) but there are
-:term:`incomplete tasks <incomplete task>` present, the scheduler will
-:term:`stall` and stay alive for 1 hour (by default) awaiting user intervention
+If there is nothing more to run, but there are
+:term:`incomplete <output completion>` tasks present in the
+:term:`n=0 window <n-window>` the workflow did not run to
+completion, so the scheduler will :term:`stall` and stay
```

Comment:
This definition isn't correct (see other comment in the glossary page).

Reply:
reworded